src/components/editor/ImportResourceTemplate.jsx

```diff
@@ -1,120 +1,123 @@
 // Copyright 2018 Stanford University see Apache2.txt for license
 
-import React, {Component} from 'react'
+import SinopiaServer from 'sinopia_server'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import Header from './Header'
 import ImportFileZone from './ImportFileZone'
 import SinopiaResourceTemplates from './SinopiaResourceTemplates'
 import UpdateResourceModal from './UpdateResourceModal'
 import { loadState } from '../../localStorage'
 import Config from '../../../src/Config'
-const SinopiaServer = require('sinopia_server')
-const instance = new SinopiaServer.LDPApi()
-const curJwt = loadState('jwtAuth')
-const _ = require('lodash')
-instance.apiClient.basePath = Config.sinopiaServerBase
-if (curJwt !== undefined) {
-  instance.apiClient.authentications['CognitoUser'].accessToken = curJwt.id_token
-}
 
 class ImportResourceTemplate extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      redirect: false,
-      group: '',
       message: [],
       createResourceError: [],
       updateResourceError: '',
       updateKey: 0,
       modalShow: false
     }
+    // Moving this into the constructor makes it easier to stub in tests
+    this.instance = new SinopiaServer.LDPApi()
+    const curJwt = loadState('jwtAuth')
+
+    this.instance.apiClient.basePath = Config.sinopiaServerBase
+    if (curJwt !== undefined) {
+      this.instance.apiClient.authentications['CognitoUser'].accessToken = curJwt.id_token
+    }
   }
 
   componentDidMount() {
-    const incrementedKey = this.state.updateKey +1
-    this.setState({updateKey: incrementedKey})
+    const incrementedKey = this.state.updateKey + 1
+    this.setState({ updateKey: incrementedKey })
   }
 
   modalClose = () => {
     this.setState( { modalShow: false } )
   }
 
-  //resource templates are set via ImportFileZone and passed to ResourceTemplate via redirect to Editor
+  // Resource templates are set via ImportFileZone and passed to ResourceTemplate via redirect to Editor
   setResourceTemplates = (content, group) => {
     const profileCount = content.Profile.resourceTemplates.length
-    content.Profile.resourceTemplates.map(rt => {
-      this.fulfillCreateResourcePromise(this.createResourcePromise(rt, group, profileCount))
+    content.Profile.resourceTemplates.forEach(async rt => {
```

Comment:
Since you are using the arrow function in this `forEach` as an anonymous function you could just as easily use the `map` operator.